DataFormats/Detectors/FIT/FDD/src/DataFormatsFDDLinkDef.h

```diff
@@ -36,5 +36,11 @@
 #pragma link C++ class o2::fdd::CTFHeader + ;
 #pragma link C++ class o2::fdd::CTF + ;
 #pragma link C++ class o2::ctf::EncodedBlocks < o2::fdd::CTFHeader, 8, uint32_t> + ;
-
+/*
+#include "DetectorsCommonDataFormats/DetID.h"
+#include "DataFormatsFIT/ChannelData.h"
+#pragma link C++ class o2::fit::ChannelData < o2::detectors::DetID::FDD> + ;
+#include "DataFormatsFIT/Digit.h"
+#pragma link C++ class o2::fit::Digit < o2::detectors::DetID::FDD> + ;
```

Comment:
this is just a comment ... should probably not be committed

Reply:
@sawenzel Hi, I turned off these in latest commits (for some reason I couldn't reproduce CICD issues on localhost). CICD takes too much time, so any cosmetics or minor hotfixes will go in next PR.